ui/components/app/confirm/shared/info/row/row.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { createContext } from 'react';
+import Tooltip from '../../../../../ui/tooltip/tooltip';
+import { Box, Icon, IconName, Text } from '../../../../../component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BorderRadius,
+  Color,
+  Display,
+  FlexDirection,
+  FlexWrap,
+  IconColor,
+  JustifyContent,
+  TextColor,
+  TextVariant,
+} from '../../../../../../helpers/constants/design-system';
+
+export enum ConfirmInfoRowVariant {
+  Default = 'default',
+  Critical = 'critical',
+  Warning = 'warning',
+}
+
+export type ConfirmInfoRowProps = {
+  label: string;
+  children: React.ReactNode | string;
+  tooltip?: string;
+  variant?: ConfirmInfoRowVariant;
+};
+
+const BACKGROUND_COLORS = {
+  [ConfirmInfoRowVariant.Default]: undefined,
+  [ConfirmInfoRowVariant.Critical]: BackgroundColor.errorMuted,
+  [ConfirmInfoRowVariant.Warning]: BackgroundColor.warningMuted,
+};
+
+const TEXT_COLORS = {
+  [ConfirmInfoRowVariant.Default]: TextColor.textAlternative,
+  [ConfirmInfoRowVariant.Critical]: Color.errorAlternative,
+  [ConfirmInfoRowVariant.Warning]: Color.warningAlternative,
+};
+
+const TOOLTIP_ICONS = {
+  [ConfirmInfoRowVariant.Default]: IconName.Question,
+  [ConfirmInfoRowVariant.Critical]: IconName.Warning,
+  [ConfirmInfoRowVariant.Warning]: IconName.Warning,
+};
+
+export const ConfirmInfoRowContext = createContext({
+  variant: ConfirmInfoRowVariant.Default,
+});
+
+export const ConfirmInfoRow = ({
+  label,
+  children,
+  variant = ConfirmInfoRowVariant.Default,
+  tooltip,
+}: ConfirmInfoRowProps) => (
+  <ConfirmInfoRowContext.Provider value={{ variant }}>
+    <Box
+      display={Display.Flex}
+      flexDirection={FlexDirection.Row}
+      justifyContent={JustifyContent.spaceBetween}
+      flexWrap={FlexWrap.Wrap}
+      backgroundColor={BACKGROUND_COLORS[variant]}
+      borderRadius={BorderRadius.SM}
+      marginTop={2}
+      marginBottom={2}
+      paddingLeft={1}
+      paddingRight={1}
```

Comment:
```suggestion
      paddingLeft={2}
      paddingRight={2}
```

left & right padding are 8px in these designs https://www.figma.com/file/CeAIcptzeaI5ciTpYBaiqr/Confirmations-DS?type=design&node-id=1-1296&mode=design&t=dNGTZAuGlPgjarSx-0

Reply:
Good catch!